### ISSUE TITLE
Searchable DNS server list

### ISSUE THREAD

**Issue by n31n** [bitbeans/SimpleDnsCrypt#476]
I would suggest, and request, a text search box in the 'Resolvers' screen to quickly sort through them.
It is sort of a hassle reading the names of all 60 servers to find the slow ones listed in the query log.

Thank you

**Comment by kaischmerer** [comment]
Good idea. Additionally an option in the main menu for presorting would be good: 
**only servers with filters**

**Comment by 3xploiton3** [comment]
![image](https://user-images.githubusercontent.com/19517680/157148174-c25ef362-0faa-40ae-8963-58b6f7299160.png)

**Comment by n31n** [comment]
Simple Dnscrypt has been forked as this has not been worked on in quite a while.

https://github.com/instantsc/SimpleDnsCrypt
